asistente_ladm_col/utils/qgis_utils.py

```diff
@@ -895,6 +895,32 @@ def copy_csv_to_db(self, csv_layer, db, target_layer_name):
 
         return True
 
+    def remove_all_layers_and_groups(self):
```

Comment:
Not needed, just use `QgsProject.instance().layerTreeRoot().removeAllChildren()`.

Reply:
It's done, thanks.